backtrace-library/src/main/java/backtraceio/library/models/BacktraceExceptionHandler.java

```diff
@@ -52,9 +55,10 @@ public void uncaughtException(final Thread thread, final Throwable throwable) {
         OnServerResponseEventListener callback = getCallbackToDefaultHandler(thread, throwable);
 
         BacktraceLogger.e(LOG_TAG, "Sending uncaught exception to Backtrace API", throwable);
-        BacktraceReport report = new BacktraceReport(this.getCausedException(throwable), BacktraceExceptionHandler.customAttributes);
-        report.attributes.put(BacktraceAttributeConsts.ErrorType, BacktraceAttributeConsts.UnhandledExceptionAttributeType);
-        this.client.send(report, callback);
+        for (BacktraceReport report :
+                this.transformExceptionIntoReports(throwable)) {
+            this.client.send(report, callback);
+        }
```

Comment:
Shouldn't this be done in an other place? What if someone sends an exception manually through `Send`?

IMO this should be done when the report is being converted into data acceptable by Backtrace.

Reply:
Planning to move it to the Client. We should also consider doing it via database instead at this point if the database is enabled